src/askui/models/openrouter/handler.py

```diff
@@ -0,0 +1,77 @@
+import os
+from typing import Type
+
+from openai import OpenAI
+from typing_extensions import override
+
+from askui.models.exceptions import QueryNoResponseError
+from askui.models.models import GetModel
+from askui.models.types.response_schemas import ResponseSchema
+from askui.utils.image_utils import ImageSource
+
+from .prompts import PROMPT_QA
+from .settings import OpenRouterSettings
+
+
+class OpenRouterGetModel(GetModel):
+    def __init__(self, settings: OpenRouterSettings):
```

Comment:
```suggestion
    def __init__(self, settings: OpenRouterSettings | None = None):
```

Would make this optional to make it more easy to use `OpenRouterGetModel` and init the `OpenRouterSettings` internally if `None`.